tests/HttpClient/UnleashHttpClientTest.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+namespace Stogon\UnleashBundle\Tests\HttpClient;
+
+use PHPUnit\Framework\TestCase;
+use Stogon\UnleashBundle\HttpClient\UnleashHttpClient;
+use Symfony\Contracts\HttpClient\Exception\ClientExceptionInterface;
+use Symfony\Contracts\HttpClient\Exception\TimeoutExceptionInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+use Symfony\Contracts\HttpClient\ResponseInterface;
+
+/**
+ * @coversDefaultClass \Stogon\UnleashBundle\HttpClient\UnleashHttpClient
+ */
+class UnleashHttpClientTest extends TestCase
+{
+	/**
+	 * @covers ::fetchFeatures
```

Comment:
Please add `@covers ::__construct`, otherwise the test coverage will mark the test as "risky"